components/ChallengeMaterial.tsx

```diff
@@ -102,6 +104,47 @@ export const ChallengeQuestionCard: React.FC<ChallengeQuestionCardProps> = ({
   const { data } = React.useContext(SessionContext)
   const username = _.get(data, 'userInfo.username', '')
   const diff = _.get(currentChallenge, 'submission.diff', false)
+  let files = null
+
+  if (diff) files = gitDiffParser.parse(diff)
```

Comment:
can this be done with the ternary operator? 
`const files = (diff) ? gitDiffParser.parse(diff) : null`